website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -90,44 +96,69 @@ function _fangornActionColumn (item, col) {
 
     if (item.kind === 'folder' && item.data.addonFullname && item.data.version === 'latest' && item.data.permissions.edit) {
         buttons.push(
-            {
-                'name' : '',
-                'tooltip' : 'Upload file',
-                'icon' : 'fa fa-upload',
-                'css' : 'fangorn-clickable btn btn-default btn-xs',
-                'onclick' : Fangorn.ButtonEvents._uploadEvent
-            },
-            {
-                'name' : '',
-                'tooltip' : 'Publish Dataset',
-                'icon' : 'fa fa-globe',
-                'css' : 'btn btn-primary btn-xs',
-                'onclick' : dataversePublish
-            }
+            { name : 'uploadFiles', template : function(){
+                return m('.fangorn-toolbar-icon.text-success', {
+                        onclick : function(event) { _uploadEvent.call(self, event, item); } 
+                    },[
+                    m('i.fa.fa-upload'),
+                    m('span.hidden-xs','Upload')
+                ]);
+            }},
+            { name : 'dataverseRelease', template : function(){
+                return m('.fangorn-toolbar-icon.text-primary', {
+                        onclick : function(event) { dataverseRelease.call(self, event, item) } 
```

Comment:
`dataverseRelease` is undefined here. Please use `jshint` to find any syntax/lint errors. You can ignore `Possible strict violation` errors.